src/Compilers/Core/Portable/xlf/CodeAnalysisResources.es.xlf

```diff
@@ -12,6 +12,11 @@
         <target state="translated">Se debe especificar un nombre de lenguaje para esta opción.</target>
         <note />
       </trans-unit>
+      <trans-unit id="ChangingVersionOfAssemblyReferenceIsNotAllowedDuringDebugging">
+        <source>Changing the version of an assembly reference is not allowed during debugging: '{0}' changed version to '{1}'.</source>
+        <target state="new">Changing the version of an assembly reference is not allowed during debugging: '{0}' changed version to '{1}'.</target>
```

Comment:
>Changing the version of an assembly reference is not allowed during debugging: '{0}' changed version to '{1}'. [](start = 28, length = 110)

Consider temporarily mangling new strings and making sure Spanish_Unit_Tests pass.